src/pickups/datastore/pickups.js

```diff
@@ -47,7 +60,9 @@ export default {
       return getters.byCurrentGroup.filter(({ store }) => store && store.isActiveStore)
     },
     byActiveStoreOneTime: (state, getters) => {
-      return getters.byActiveStore.filter(e => !e.series)
+      return getters.byActiveStore
+        .filter(e => !e.series)
+        .filter(p => !p.hasStarted)
```

Comment:
A comment could be helpful to understand why started pickups are only filtered at this stage and not elsewhere.